### PR TITLE
Block Domain - Generic new playbook and subplaybooks

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/39039

## Description
Aligned with our block indicator playbooks.
This playbook will provide the ability to block domain name.

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/FireEyeEX/Playbooks/playbook-Block_Domain_-_FireEye_Email_Security.yml [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+inputs:
+- key: Domain
+  value: {}
+  required: false
+  description: The Domain to block.
+  playbookInputQuery:
+outputs: []
+tests:
+- No tests (auto formatted)
+fromversion: 6.0.0
```
Comment (ShahafBenYakir) [inline Packs/FireEyeEX/Playbooks/playbook-Block_Domain_-_FireEye_Email_Security.yml:218]: @melamedbn this is the bug

File: Packs/ProofpointThreatResponse/.pack-ignore [hunk 2]
```diff
@@ -1,2 +1,5 @@
 [file:ProofpointThreatResponse.yml]
 ignore=IN126
+
```
Comment (altmannyarden) [inline Packs/ProofpointThreatResponse/.pack-ignore:3]: What is that ignore?

File: Packs/ProofpointThreatResponse/.pack-ignore [hunk 3]
```diff
@@ -1,2 +1,5 @@
...
+[file:README.md]
+ignore=RM104
```
Comment (altmannyarden) [inline Packs/ProofpointThreatResponse/.pack-ignore:5]: What is that ignore?

File: Packs/TrendMicroApex/.pack-ignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+[file:README.md]
+ignore=RM104
```
Comment (altmannyarden) [inline Packs/TrendMicroApex/.pack-ignore:2]: What is that ignore?

File: Packs/Zscaler/.pack-ignore [hunk 5]
```diff
@@ -1,2 +1,2 @@
 [file:README.md]
-ignore=RM106
\ No newline at end of file
+ignore=RM104
```
Comment (altmannyarden) [inline Packs/Zscaler/.pack-ignore:2]: You wanted to add or remove the existing?

File: Packs/ProofpointThreatResponse/.pack-ignore (hunk 3 above)
Comment (melamedbn) [reply Packs/ProofpointThreatResponse/.pack-ignore:5]: empty readme

File: Packs/TrendMicroApex/.pack-ignore (hunk 4 above)
Comment (melamedbn) [reply Packs/TrendMicroApex/.pack-ignore:2]: empty readme

File: Packs/Zscaler/.pack-ignore (hunk 5 above)
Comment (melamedbn) [reply Packs/Zscaler/.pack-ignore:2]: Didn't get what you mean

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_Indicators_-_Generic_v2_5_5_README.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+| CheckpointFWRule.Domain.Type | Rule domain type. | unknown |
+| CheckpointFWRule.Hits.FirstDate | The date of the first hit for the rule. | unknown |
+| CheckpointFWRule.Hits.LastDate | The date of the last hit for the rule. | unknown |
+| CheckpointFWRule.Hits.Level | Level of rule hits. | unknown |
+| CheckpointFWRule.Hits.Percentage | Percentage of rule hits | unknown |
+| CheckpointFWRule.Hits.Value | Value of rule hits. | unknown |
+
+## Playbook Image
+---
+![Block Indicators - Generic v2](https://raw.githubusercontent.com/demisto/content/1d674c76b038bef0467d6acb53cee697e7c55e45/Packs/CommonPlaybooks/doc_files/Block_Indicators_-_Generic_v2.png)
```
[the remark this answers is not in the document]
Comment (melamedbn) [reply Packs/CommonPlaybooks/Playbooks/playbook-Block_Indicators_-_Generic_v2_5_5_README.md]: Fixed

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**